enhancements/network/ovn-secondary-networks.md

```diff
@@ -0,0 +1,332 @@
+---
+title: ovn-secondary-networks
+authors:
+  - "@maiqueb"
+reviewers:
+  - "@dcbw"
+  - "@dougbtv"
+  - "@squeed"
+  - "@trozet"
+  - "@abhat"
+  - "@alexanderConstantinescu"
+  - TBD
+approvers:
+  - TBD
+creation-date: 2021-06-15
+last-updated: yyyy-mm-dd
+status: provisional
+---
+
+# OVN secondary networks
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+This enhacement proposal studies and describes how to support OVN as the
+networking provider for a Multus secondary interface.
+
+## Motivation
+The networking plugins that can be used as secondary interfaces are currently
+quite simple, and are unable to provide advanced networking requirements.
+
+Furthermore, virtualization users are used to being able to define isolated
+tenant networks that form a single L2 domain. OVN excels at this, by creating
+simple L2 overlays without the need for a network administrator.
+
+### Goals
+- create flat layer 2 overlays
+- connect a newly created pod to these overlay networks
+- these overlays can optionally be a subnet
+- IPv6 support is welcome (not an immediate priority)
+- optional connection to the external networks
+
+### Non-Goals
+- network policy concerns. There isn't a requirement for having network
+  policies for the secondary networks. There is an
+  [enhancement proposal](enhancements/network/multi-networkpolicy.md) about
+  this subject, not scoped around OVN.
+- ingress into the OVN secondary networks. This is not a current objective,
+  and should be treated in a separate enhancement proposal, which would depend
+  on the [services on secondary networks feature](TODO: add link to PR) availability.
+
+## Proposal
+
+### User Stories
+
+#### Isolated tenant network
```

Comment:
This user-story is a bit too low-level. Can you provide a user-story that describes what the end-user themselves would like? Then, you later indicate in your design how this user-story is served.

Something like "I have multiple kubevirt VM-pods, and I would like layer-2 connectivity between them. This could be used for X, Y, or Z"